deploy/helm/sumologic/templates/logs/collector/otelcol/serviceaccount.yaml

```
@@ -6,4 +6,10 @@ metadata:
   labels:
     app: {{ template "sumologic.labels.app.logs.collector.serviceaccount" . }}
     {{- include "sumologic.labels.common" . | nindent 4 }}
+  annotations:
+{{- if .Values.sumologic.logs.collector.otelcloudwatch.enabled }}
+{{- if .Values.sumologic.logs.collector.otelcloudwatch.roleArn }}
+    eks.amazonaws.com/role-arn: {{ .Values.sumologic.logs.collector.otelcloudwatch.roleArn }}
+{{- end }}
+{{- end }}
```

we can use `and` instead of two conditions:

```suggestion
  annotations:
{{- if and .Values.sumologic.logs.collector.otelcloudwatch.enabled .Values.sumologic.logs.collector.otelcloudwatch.roleArn }}
    eks.amazonaws.com/role-arn: {{ .Values.sumologic.logs.collector.otelcloudwatch.roleArn }}
{{- end }}
```